src/generic/notifmsgg.cpp

```diff
@@ -307,6 +308,7 @@ void wxNotificationMessageWindow::OnNotificationMouseLeave(wxMouseEvent& WXUNUSE
 void wxNotificationMessageWindow::OnCloseClicked(wxCommandEvent& WXUNUSED(event))
 {
     wxCommandEvent evt(wxEVT_NOTIFICATION_MESSAGE_DISMISSED);
+    evt.SetExtraLong(2);
```

Comment:
Same as above.

Reply:
Looks like I forgot to modify under `generic/`.